app/src/androidTest/java/com/android/sample/screen/MainPageTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.android.sample.screen
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTouchInput
+import androidx.compose.ui.test.swipeLeft
+import androidx.compose.ui.unit.dp
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.navigation.Route
+import com.android.sample.ui.swipePage.ReceipeDisplay
+import com.android.sample.ui.swipePage.SwipePage
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+
+@RunWith(AndroidJUnit4::class)
+class MainPageTest : TestCase() {
```

Comment:
### Review 3: Test Filename

**Review Comment:**
The test file is named `MainPageTest`, but since the `MainPage` has been replaced with `SwipePage`, it would be more consistent to rename the file to `SwipePageTest`.

Reply:
You're right, thank you! I'll refactor it.